webapp/src/main/resources/public/js/stores/PaginatorStore.js

```diff
@@ -34,7 +34,9 @@ class PaginatorStore {
     }
 
     changeCurrentPageNumber(currentPageNumber) {
-        this.currentPageNumber = currentPageNumber;
```

Comment:
why is this added?

Reply:
There was a console warning that the prop was expecting a string but got number (or vice versa). This ensures that the expected type will be received